github/github.go

```diff
@@ -880,6 +880,32 @@ func cloneRequest(r *http.Request) *http.Request {
 	return r2
 }
 
+// formatRateLimitResetDuration formats d to look like "[rate reset in 2s]" or
```

Comment:
Since you've renamed the function, this comment should also be updated to begin with `// formatRateReset formats d to ...`.